src/components/_common/overviewFilter/OverviewFilter.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from 'react';
+import { Area } from 'types/areas';
+import styles from './OverviewFilter.module.scss';
+import { translator } from '../../../translations';
+import { usePageConfig } from 'store/hooks/usePageConfig';
+import { Heading, Tag } from '@navikt/ds-react';
+import classNames from 'classnames';
+
+interface OverviewFilterProps {
+    filterUpdateCallback: (filters: Area[]) => void;
+}
+
+export const OverviewFilter = ({
+    filterUpdateCallback,
+}: OverviewFilterProps) => {
+    const area = Object.values(Area).map((area) => area);
```

Comment:
Map-kallet kan droppes (og dette arrayet kan defineres utenfor komponenten)

Reply:
Feilstavet konstant. Det skulle være "areas" i flertall, men jeg har kalt den "filterableAreas". For å nappe ut alle areas fra en enum må jeg såvidt jeg vet mappe på Object.values?

Reply:
Skal holde å kjøre Object.values på enumen for å få ut verdiene. map(n => n) lager bare en ny kopi av det arrayet igjen.